decidim-dev/lib/tasks/test_app.rake

```diff
@@ -15,4 +15,18 @@ namespace :decidim do
       )
     end
   end
+
+  desc "Generates a dummy app for testing in external installations"
+  task :generate_external_test_app do
+    dummy_app_path = File.expand_path(File.join(Dir.pwd, "spec", "decidim_dummy_app"))
+
+    Bundler.with_clean_env do
```

Comment:
This `Bundler.with_clean_env` block should go away, it's important so that the external app can really use its main `Gemfile`.